core/utils/poly-look/src/react-components/tabs/tabs.jsx

```diff
@@ -36,9 +37,10 @@ const Tabs = ({ children, swipe = true }) => {
         slidesPerView={1}
         initialSlide={0}
         watchOverflow={true}
-        onSlideChange={(swiper) =>
-          setActiveTabId(children[swiper.activeIndex].props.id)
-        }
+        onSlideChange={(swiper) => {
+          setActiveTabId(children[swiper.activeIndex].props.id);
+          onTabChange(children[swiper.activeIndex].props.id);
```

Comment:
maybe define this above `const handleSlideChange({ activeIndex }) => { const id = children[activeIndex].props.id ;  setActiveTabId(id); onTabChange(id)} ` and use it there

Reply:
yeah that's a good point, but since it is being called in OnTabChange and the arguments there are slightly different, it might be best to keep the two functions separate